app/dashboard/views.py

```diff
@@ -5344,3 +5347,92 @@ def bulkDM(request):
     }
 
     return TemplateResponse(request, 'bulk_DM.html', context)
+
+
+def validate_number(user_id, twilio, phone, redis):
+    validation = twilio.lookups.phone_numbers(phone).fetch(type=['caller-name', 'carrier'])
+
+    if validation.caller_name and validation.caller_name["caller_type"] == 'CONSUMER':
+        return JsonResponse({
+            'success': False,
+            'msg': 'Only support consumer numbers'
+        }, status=401)
+    if validation.carrier and validation.carrier['type'] != 'mobile':
+        return JsonResponse({
+            'success': False,
+            'msg': 'Phone type isn\'t supported'
+        }, status=401)
+
+    verification = twilio.verify.verifications.create(to=phone, channel='sms')
+    redis.set(f'verification:{user_id}:timestamp', timezone.now().isoformat())
+    redis.set(f'verification:{user_id}:phone', phone)
+
+
+
+@login_required
+def send_verification(request):
+    phone = request.POST.get('phone')
+    redis = RedisService().redis
+    twilio = TwilioService()
+    has_previous_validation = redis.get(f'verification:{request.user.id}:timestamp')
```

Comment:
might be worth storing this in the DB instead of on redis.  that seems like a better place to store it, as it doesnt benefit from being in redis (its not written often, so having it in memory is pretty expensive)